lab_5_scrapper/scrapper_config.json

```diff
@@ -1,9 +1,12 @@
 {
-    "seed_urls": [],
-    "headers": {},
-    "total_articles_to_find_and_parse": 0,
-    "encoding": "",
-    "timeout": 0,
+    "seed_urls": ["https://3dnews.ru/news", "https://3dnews.ru/news/page-2.html", "https://3dnews.ru/news/page-3.html", "https://3dnews.ru/news/page-4.html"],
+    "headers": {"cookie": "stg_traffic_source_priority=1; stg_last_interaction=Mon%2C%2008%20Apr%202024%2009:21:18%20GMT; stg_returning_visitor=Mon%2C%2008%20Apr%202024%2009:21:18%20GMT",
+           "accept-Language": "ru-RU,ru;q=0.9,en-US;q=0.8,en;q=0.7",
+           "accept": "text / html, application / xhtml + xml, application / xml; q=0.9, image / avif, image / webp, image / apng, * / *;q=0.8, application / signed - exchange; v=b3; q=0.7",
+           "user-agent": "Mozilla / 5.0(Macintosh; Intel Mac OS X 10_15_7) AppleWebKit / 537.36(KHTML, like Gecko) Chrome / 123.0.0.0 Safari / 537.36"},
+    "total_articles_to_find_and_parse": 100,
```

Comment:
crawler checks failed because of timeout -> 

i suggest you should lower the number of articles to 2-5

Reply:
tried and also changed the headers, they were the problem I think